examples/gameroom/database/tables/gameroom_tables.sql

```diff
@@ -12,14 +12,6 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
--- Create tables
```

Comment:
Maybe we should add a migration for this change?

Reply:
That's a good question. I don't think it's a trivial migration since we'd also need to supply a way of porting existing users from the old database to biome, in addition to providing the migration files in order to do an actual migration. Just adding the migration files seems weird since you'd just be deleting a preexisting users, and adding a script also seems weird since we'd be migrating users for an app that plays tic-tac-toe.